sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -71,7 +64,7 @@
   </SALTelemetry>
   <SALTelemetry>
     <EFDB_Topic>ESS_airFlow</EFDB_Topic>
-    <Description>AirFlow Sensor - Windsonic1-L (Bi-axial).</Description>
+    <Description>2-dimensional anemometer data.</Description>
```

Comment:
Maybe make the description in line with the one at line 7? for instance `Air flow measured by a 2-dimensional anemometer.`